charts/alloy-psql/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
Seems like if you are running psql in containers, you only need a single `postgres` chart?  Or none at all, since I assume there is one in the helm chart repo?